config/default/uids_base.settings.yml

```diff
@@ -15,6 +15,8 @@ header:
   sticky: 0
   toppage: 0
   top_links_limit: 2
+style:
+  style-selector: on-brand
```

Comment:
Not sure why the `font-family` setting used a dash instead of an underscore, but lets stick with the underscore for naming config properties.

Reply:
```suggestion
  style_selector: brand
```